datafusion-cli/src/print_options.rs

```diff
@@ -127,9 +107,125 @@ impl PrintOptions {
         Ok(())
     }
 
+    pub async fn print_table_batch(
```

Comment:
Could you add some comments here, as well as how it is different than `print_batches`